src/main/java/org/stellar/sdk/ManageBuyOfferOperation.java

```diff
@@ -124,13 +123,25 @@ public Builder(
         @NonNull Asset selling,
         @NonNull Asset buying,
         @NonNull String amount,
-        @NonNull String price) {
+        @NonNull Price price) {
       this.selling = selling;
       this.buying = buying;
       this.amount = amount;
       this.price = price;
     }
 
+    /** An alias for {@link Builder#Builder(Asset, Asset, String, Price)} */
+    public Builder(
+        @NonNull Asset selling,
+        @NonNull Asset buying,
+        @NonNull String amount,
+        @NonNull String price) {
+      this.selling = selling;
```

Comment:
same overloaded constructor pattern could be used here, convert and invoke the main constructor instead:

```
this(selling, buying, amount, Price.fromString(price))

```